client/src/document/organisms/toc/index.scss

```diff
@@ -5,7 +5,6 @@
 .document-toc {
   margin-bottom: 2rem;
   padding: 0;
-  position: relative;
```

Comment:
Test what this changes?

Reply:
Tested with all screen sizes: It doesn't make any difference, so we can safely remove it.